.github/workflows/app-build-and-deploy.yml

```diff
@@ -97,20 +97,20 @@ jobs:
   build-and-publish-image:
     name: Build and publish Docker image
     needs: get-version
-    uses: clearlydefined/operations/.github/workflows/app-build-docker-image.yml@v2.0.0
+    uses: ./.github/workflows/app-build-docker-image.yml
```

Comment:
This is a really nice change.  Not having to update the version on every sub-workflow call is going to greatly improve long term maintenance.